pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_1.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+    </data>
+    <data>
+    </data>
+</odoo>
```

Comment:
@dreispt 
The case when `<data noupdate="0">` is equal to only the tag `<data>`
The default valud of `noupdate` is `0`
Therefore I add only the `<data>` tag

Are you agree?